src/averageAges.js

```diff
@@ -55,7 +73,19 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const womenWithChil = people
```

Comment:
I th u shouldn mak shortc li tha. Not easy to read, right? 😄 

What I meant is that you shouldn't make shortcuts like that. You should make your variable names maximally clear. What about trying to do it like that:
```suggestion
  const mothers = people
```